source/Sashimi.Terraform/Validation/TerraformDeploymentActionValidation.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentValidation;
+using Sashimi.Server.Contracts.ActionHandlers.Validation;
+using Sashimi.Server.Contracts.CloudTemplates;
+
+namespace Sashimi.Terraform.Validation
+{
+    public class TerraformDeploymentActionValidation : IDeploymentActionValidator
```

Comment:
Just thinking, what if we made `IDeploymentActionValidator` implement a Validator? then we don't need this boilerplate?

```
public interface IDeploymentActionValidator : IValidator<DeploymentActionValidationContext>
{
}

public class TerraformValidator : AbstractValidator<DeploymentActionValidationContext>, IIDeploymentActionValidator 
{
}
```

Reply:
We should raise it with @slewis74